mass_flask_webui/forms/request_analysis.py

```diff
@@ -0,0 +1,9 @@
+from flask_wtf import Form
+from wtforms import SelectField, IntegerField, SubmitField
+from wtforms.validators import InputRequired
+
+
+class RequestAnalysisForm(Form):
+    analysis_system = SelectField('Analysis system', choices=[], validators=[InputRequired()])
```

Comment:
Is there a particular reason why we would not query the available analysis systems in this object?